scrapy_quintain_utils/pipelines/screenshot.py

```diff
@@ -26,21 +32,16 @@ def from_settings(cls, settings):
         s3store.POLICY = settings['IMAGES_STORE_S3_ACL']
         api_key_splash = settings['API_KEY']
         store_uri = settings['IMAGES_STORE']
-        return cls(store_uri, s3store, api_key_splash, settings=settings)
+        splash_args = settings.get('SCREENSHOT_PIPELINE_SPLASH_ARGS', cls.DEFAULT_SPLASH_ARGS)
```

Comment:
Allow to use splash args for screenshot pipeline in settings. Otherwise, use default one for backwards compatibility